force-app/main/default/translations/pt_BR.translation-meta.xml

```diff
@@ -1237,7 +1237,7 @@
     </customLabels>
     <customLabels>
         <name>RD2_EntryFormUpdateSuccessMessage</name>
-        <label>A Doação recorrente "{0}” foi salva.</label>
+        <label>A Doação recorrente &quot;{0}” foi salva.</label>
```

Comment:
Same as above.

Reply:
This should be changed to an entity as well.